docker/Dockerfile

```diff
@@ -10,7 +10,7 @@
 ### The date in the Server command must either match, or be more recent than,
 ### the date in the FROM command
 ### Date format is ISO year/month/day
-FROM archlinux:base-20210829.0.32635
+FROM archlinux:base-20210829.0.32635 as base
```

Comment:
What's the selling point of changing the dockerfile like this? It seems to me that it makes it a bit harder to follow, but not terribly so. Perhaps enough to counter the somewhat faster dockerfile builds when actively working with this library, but I am not sold either way.

Reply:
I had it setup that way because I ways building multiple parallel dependencies. At the moment now the case is less compelling because Kaldi and OpenFST are the only ones that take a while to compile and Kaldi depends on OpenFST.

I've been thinking about adjusting the voice2json installation section to follow the conventions established in the docker file however I fear that I won't have enough time at the moment to improve this and I feel that the improvement is readably. Specifically, I was thinking of adding a section to the `install-from-source` script that can handle `makepkg` dependencies and then installing them in the toolchains sections.